include/oneapi/dpl/experimental/kt/esimd_radix_sort_utils.h

```diff
@@ -436,6 +436,100 @@ VectorStore(uint32_t offset,
+
+    // mask, simd_mask<N>:
+    // +----------------------------------------------------------+-----------------------------------------------------------+-----+----------------------------------------------------------+
+    // |                             m0                           |                             m1                            | ... |                             mN                           |
+    // +----------------------------------------------------------+-----------------------------------------------------------+-----+----------------------------------------------------------+
+    //                                                                                          |
+    // __uint32_mask, simd_mask<2 * N>:                                                         V
+    // +-----------------------------+----------------------------+-----------------------------+-----------------------------+-----+-----------------------------+----------------------------+
+    // |             m0              |             m0             |             m1              |             m1              | ... |             mN              |             mN             |
+    // +-----------------------------+----------------------------+-----------------------------+-----------------------------+-----+-----------------------------+----------------------------+
+
+    using TypeX32 = decltype(DestTypeResolver(T{}));
+
+    // https://intel.github.io/llvm-docs/doxygen/classsycl_1_1__V1_1_1ext_1_1intel_1_1esimd_1_1detail_1_1simd__obj__impl.html#ad1eaca7ca43fcf87a29a9714b93d8109
+    // bit_cast_view
+    sycl::ext::intel::esimd::simd<TypeX32, 2 * N> __x32_vals = vals.template bit_cast_view<TypeX32>();
+
+    __ESIMD_NS::simd<uint32_t, 2 * N> __x32_offsets;
+    __ESIMD_NS::simd_mask<2 * N>      __x32_mask(0);
+
+    // TODO required to implement this through simd/mask operations
+    // simd select stride
+    #pragma unroll
+    for (size_t src_index = 0; src_index < N; ++src_index)
+    {
+        __x32_offsets[2 * src_index    ] = offsets[src_index];
+        __x32_offsets[2 * src_index + 1] = offsets[src_index] + sizeof(TypeX32);
+
+        __x32_mask[2 * src_index    ] = mask[src_index];
+        __x32_mask[2 * src_index + 1] = mask[src_index];
+    }
+
+    // https://intel.github.io/llvm-docs/doxygen/group__sycl__esimd__memory__slm.html#gac6d34e476b1addadb3f27648d62322a4
+    //template<typename T , int N>
+    //__ESIMD_API std::enable_if_t<(N == 1 || N == 8 || N == 16 || N == 32) && (sizeof(T) <= 4)>
+    //sycl::_V1::ext::intel::esimd::slm_scatter(
+    //    simd<uint32_t, N> offsets,                    // Byte offsets within the SLM of each element.
+    //    simd<T, N> vals,                              // Values to be written.
+    //    simd_mask<N> mask = 1);                       // Operation mask. All-1 by default.
```

Comment:
This comment looks excessive. Could you remove it?

Reply:
Fixed (removed)